mm/kasan/kasan.h

```diff
@@ -20,6 +36,25 @@ struct access_info {
 	unsigned long ip;
 };
 
+struct asan_source_location {
```

Comment:
We don't yet have names prefixed with 'asan_' in kernel. I think it's better to rename it to 'kasan_' to keep it consistent with the rest of the names here, even if it diverges from user-space naming.